docs/source/deployment/aws_ecs.rst

```diff
@@ -130,8 +159,11 @@ Create AWS ECR repository
 
 .. code-block:: bash
 
-    > cd /Users/bozhaoyu/bentoml/repository/IrisClassifier/20200121141808_FE78B5
-    > docker build . --tag=192023623294.dkr.ecr.us-west-2.amazonaws.com/irisclassifier-ecs
+    # Download and install jq, the JSON processor: https://stedolan.github.io/jq/download/
```

Comment:
`JSOn processor` is a bit inaccurate, let's use `command-line JSON processor`:

# Install jq, the command-line JSON processor: https://stedolan.github.io/jq/download/